webpack/assets/images

```diff
@@ -0,0 +1 @@
+../../app/assets/images/
```

Comment:
IIRC this is so when we want to add the webpack dev server it will also have access to the images, not sure if it's required at this point